backend/census_historical_migration/admin.py

```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin  # noqa: F401
 
-# Register your models here.
+from census_historical_migration.models import (
+    ELECAUDITHEADER,

```

Comment:
Any reason why `ELECAUDITHEADER` is part of this PR ?

Reply:
This is inherited from the c2g branch.  Tests for other tables need to be added in future.